### ISSUE TITLE
Web front end/usability improvements for Junar file uploader

### ISSUE THREAD

**Issue by howdoicomputer** [codeforsanjose/Project-Ideas#51]
The City of San Jose uses Junar as its platform of choice for presenting data to civic hacktivists. 

Unfortunately, it seems like Junar has had a a poor rate of adoption within the city itself. There is a large discrepancy when comparing the amount of data stored in Palo Alto's platform to the data stored in San Jose's platform. For the below links, you can explore the available dashboards by clicking on the tabs.

http://data.cityofpaloalto.org/dashboards/8853/explore-palo-alto/
http://data.sanjoseca.gov/dashboards/9386/transparency-and-accountability/

Going through Junar's GitHub repositories, I found the tool that's used to do bulk CSV/Excel uploads to hosted data portals.

https://github.com/Junar/junar-uploader

Using a command line tool isn't the most user friendly activity in the world for a potentially non-technical government employee. Maybe create a single page, small, web front end for uploading a CSV to their API?

This would need @mthong's weigh in on whether or not this is a valid project. A developer picking up this project would need to work with the Junar developers, most likely.

**Comment by mthong** [comment]
Interesting proposal. Junar provides a web-based GUI dashboard for city staff to upload datasets, but I don't think it does bulk uploads. I'll ask the Open Data Architect whether she thinks this would be a meaningful, valuable tool for city staff.

**Comment by howdoicomputer** [comment]
@mthong Yeah, if the architect comes back with "we're all good here" then I'm glad to hear it. However, if there are some gains to be had for improving UX for data uploading then it might be worth making it a volunteer effort. Outside of looking at GitHub repos and possibly outdated Junar docs, I don't have much visibility into the data upload process so it could be super awesome and this ticket could be totally invalid.
